### ISSUE TITLE
[Assistive tech and device support] Page functionality isn't keyboard accessible. (11.05.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#33877]
#### VFS team name
AFS Education

#### VFS product name
GI Bill Comparison Tool

#### Point of Contact/Reviewers
Noah Gelman (@noahgelman) - Accessibility

---

### Platform Issue
Page functionality isn't keyboard accessible.

### Issue Details
Sidebar "Learn more" modal content is not accessible by keyboard/screenreader technology. Focus is brought to the close button and if there is no link present in the modal content then the user is unable to navigate the content of the modal with the keyboard and therefore cannot read the modal content with a screenreader. 

### Link, screenshot or steps to recreate
Navigate to GI Bill Search tool results. Use keyboard to tab to a "Learn More" button in sidebar. Open modal with keyboard. Try and use arrow keys or tabbing to reach the mooal content

### VA.gov Experience Standard
[Category Number 11, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.1.1_A

---

### Platform Recommendation
Upon opening the "Learn More" modal focus should moved to the modal header so the user can navigate through the content with the keyboard. The user can still tab to any links or the close button. The header should also be added to the tabindex so the user can tab back to the content as well.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by hallihocine** [comment]
Ticket has been validated in staging: Sidebar "Learn more" modal content is now accessible by keyboard/screenreader technology. The user can tab to any link if present in the content, or the to the close button and can tab back to the content as well.

https://user-images.githubusercontent.com/86266024/146576571-edb87eac-b034-482a-8f52-8ce9a22c6b82.mov

### **Screenshots:** 
- Focus on Modal header: 
<img width="1054" alt="Focus on modal header" src="https://user-images.githubusercontent.com/86266024/146576835-a61f8e53-1498-44a9-b944-95c713bf3ff7.png">
- Focus on link: 
<img width="1010" alt="Focus on link" src="https://user-images.githubusercontent.com/86266024/146576875-71bc515d-8884-48a5-9181-26afa3dddcf4.png">
- Focus on close button: 
<img width="940" alt="Focus on close button" src="https://user-images.githubusercontent.com/86266024/146576964-44924297-3c72-4187-a908-39838e333348.png">

**Comment by hallihocine** [comment]
- Test Run in Test Tail: 
https://dsvavsp.testrail.io/index.php?/runs/view/2740&group_by=cases:section_id&group_order=asc&group_id=3226
- Test Script/Steps: 
[[Assistive tech and device support] Page functionality isn't keyboard accessible. (11.05.1).xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/7736513/Assistive.tech.and.device.support.Page.functionality.isn.t.keyboard.accessible.11.05.1.xlsx)

**Comment by ERRonat0609** [comment]
@mattpointzxer0 This issue is ready for your review/approval. Please close upon validation. Thanks!!

**Comment by mattpointzxer0** [comment]
👍
